src/modules/LanItems.rb

```diff
@@ -1522,11 +1531,11 @@ def BuildLanOverview
         conn = HTML.Bold(format("(%s)", _("Not connected"))) if !item_hwinfo["link"]
         conn = HTML.Bold(format("(%s)", _("No hwinfo"))) if item_hwinfo.empty?
 
-        mac_dev = HTML.Bold("MAC : ") + item_hwinfo["mac"].to_s + "<br>"
+        mac_dev = HTML.Bold("MAC : ") + item_hwinfo["permanent_mac"].to_s + "<br>"
         bus_id  = HTML.Bold("BusID : ") + item_hwinfo["busid"].to_s + "<br>"
         physical_port_id = HTML.Bold("PhysicalPortID : ") + physical_port_id(ifcfg_name) + "<br>"
 
-        rich << " " << conn << "<br>" << mac_dev if IsNotEmpty(item_hwinfo["mac"])
+        rich << " " << conn << "<br>" << mac_dev if IsNotEmpty(item_hwinfo["permanent_mac"])
```

Comment:
NP: i know it is a merge but may be that we can start droping these ```IsNotEmpty```helpers

Reply:
network-ng will kill it :P